applications/sim_showers_for_trigger_rates.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/python3
+
+'''
+    Summary
+    -------
+    This application simulates showers to be used in trigger rate calculations.
+    Arrays with one (1MST) or four telescopes (4LST) can be used, in case of \
+    mono or stereo trigger configurations, respectively. 
+
+    Simulations are managed by the shower_simulator module.
+    Each run is simulated in a job. Each job is submitted by using the submission \
+    command from the global config settings (see config_template.yml). \
+    The config entry extraCommands can be used to extra commands to be ran in each job,
+    before the actual simulation.
+
+    At the moment, the shower simulations are performed by CORSIKA, which requires \
+    the zstd package. Please, make sure that the command to set your zstd path is \
+    properly set by the extraCommands in config.yml.
+
+    Command line arguments
+    ----------------------
+    array (str, required)
+        Name of the array (1MST, 4LST ...).
+    site (str, required)
+        South or North.
+    primary (str, required)
+        Name of the primary particle (proton, helium ...).
+    nruns (int, optional)
+        Number of runs to be simulated (default=100).
+    nevents (int, optional)
+        Number of events simulated per run (default=100000).
+    zenith (float, optional)
+        Zenith angle in deg (default=20).
+    azimuth (float, optional)
+        Azimuth angle in deg (default=0).
+    output (str, optional)
+        Path of the directory to store the output simulations. By default, \
+        the standard output directory will be used.
```

Comment:
'standard output directory' - that is the standard set where, in config.yml? And is it outputLocation? Maybe write this clearer.

Reply:
It's not exactly the directory given in config.yml. We have a system to create the 'standard' output directories where the parent directory is outputLocation and the structure of the sub-directories follow a certain pattern.

The idea is that this pattern is basically uncoupled and can be changed in the future, if we find necessary. So, it doesn't make sense to write here exactly the path of the standard output because that may change.

This system for output directories is widely used over the package. It's expected that a familiar user will understand what that means.

By now I just replaced the text by 'standard output directory defined by config will be used'. I still have to figure out good terms to refer to the global config system and make all the documentation consistent.